### PR TITLE
Serve erizo.js from erizo controller

### PR DESCRIPTION
**Pull request by caboteria**
There are 2 commits here: one makes the build process build both erizo.js and erizo-debug.js so clients can ask for one or the other at any time.  The other commit makes erizoController server erizo.js (and its debug version) so clients don't need to bundle erizo.js with their code.

### CODE REVIEW

File: extras/basic_example/public/index.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <html>
   <head>
     <title>Licode Basic Example</title>
-    <script type="text/javascript" src="erizo.js"></script>
+    <script type="text/javascript" src="http://192.168.33.99:8080/erizo.js"></script>
```
Comment (MorikoHandford) [inline extras/basic_example/public/index.html:4]: I feel like I should know what IP address this is . . . but I don't.  I know it is a private network address.  So, not knowing what this IP is, is it okay that it is hard-coded?

Comment (caboteria) [inline extras/basic_example/public/index.html:4]: That's the IP that we use for the webrtc vagrant.  It's hard-coded in vagrantfile so I think it's OK to hard-code here.

Comment (MorikoHandford) [inline extras/basic_example/public/index.html:4]: Oh, I remember now.  I've made a note in the wiki to help remind me the next time I come across that IP.

**Comment by MorikoHandford** [comment]
:+1:

**Pull request merged**